app/firebase/functions/src/platforms/nanopub/create.nanopub.ts

```diff
@@ -38,6 +43,10 @@ export const createNanopublication = async (
   const originalPlatformPost = post.mirrors.find(
     (platformPost) => platformPost.platformId === PLATFORM.Twitter
   )?.posted;
+  const postedNanopub = post.mirrors.find(
+    (platformPost) => platformPost.platformId === PLATFORM.Nanopub
+  )?.posted;
+  const originalNanopubUri = postedNanopub?.post_id; // confirm if this is latest or root and if it includes the domain or just the nanopub hash
```

Comment:
@pepoospina from what I could tell, when a post is edited then republished, nothing needs to change. If it has already been published, then the `originalNanopubUri` will exist and will get added as in the supersedesOptions